client/components/feature-modules/entity/components/modals/type/delete-definition-modal.tsx

```
@@ -0,0 +1,253 @@
+                                <RadioGroup
+                                    value={deleteAction}
+                                    onValueChange={(value) =>
+                                        setDeleteAction(value as DeleteAction)
+                                    }
+                                >
+                                    {Object.entries(DELETE_ACTION_LABELS).map(([action, info]) => (
+                                        <div
+                                            key={action}
+                                            className="flex items-start space-x-3 space-y-0 rounded-md border p-4 hover:bg-accent/50 transition-colors cursor-pointer"
+                                            onClick={() => setDeleteAction(action as DeleteAction)}
+                                        >
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)